.circleci/config.yml

```diff
@@ -667,6 +740,62 @@ workflows:
           filters:
             branches:
               only: master
+
+  postgres_build:
+    jobs:
```

Comment:
We can make it `manual_approval` too, like oracle.

Reply:
Yeah that would be nice 😄 Because many times it is not necessary 👍 
@hallelujah @gsaslis What do you think?

Reply:
I'd say it depends how much confidence we have on the postgresql build staying green when we don't run the tests on it. in my experience so far, this kind of confidence builds up over time (the longer you see a build staying green, the higher the confidence) and with postgresql being a newly-supported db here, i'd err on the side of caution, till we get a better feel of what kinds of changes would break its build. 

I'd also want to avoid a situation where we skip these tests too often, (the problem with leaving some kind of testing at the "last minute" before merging to master). 

having said that, the more our compatibility matrix grows, the more it makes financial sense to move these kinds of tests outside the "Commit Phase" test suite. 

There must be other teams who have somehow addressed this matrix compatibility problem. Perhaps we can ask around and learn from them (e.g. at what point in lifecycle do they run those tests)?

Reply:
I have no confidence at all on Oracle been green, I'd like to run Oracle at least once a day. Ideally anything always ... We decided to run it manually to save resources because it is too long. 

For Postgres, I would say we should always run it.

Reply:
Anyway we should run those jobs (oracle included) nightly

Reply:
We are currently speaking about this in Slack, it is to be decided... 😄

Reply:
OK lets make it manual_approval @guicassolato

Reply:
Done